src/Network/Network/AzureFirewall/NetworkRule/NewAzureFirewallNetworkRuleCommand.cs

```diff
@@ -43,11 +44,17 @@ public class NewAzureFirewallNetworkRuleCommand : NetworkBaseCmdlet
         public string[] SourceAddress { get; set; }
 
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
             HelpMessage = "The destination addresses of the rule")]
         [ValidateNotNullOrEmpty]
         public string[] DestinationAddress { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The destination FQDNs of the rule")]
+        [ValidateNotNullOrEmpty]
+        public string[] DestinationFqdns { get; set; }
```

Comment:
Cmdlet parameters should be in singular form

Reply:
Ok I will change this and commit along with test recordings.